src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -606,13 +577,77 @@ function New-AzConnectedKubernetes {
         else {
             $PSBoundParameters.Add('AgentPublicKeyCertificate', $AgentPublicKey)
         }
+
+        # Process the Arc agentry settings and protected settings
+        # Create any empty array of IArcAgentryConfigurations.
+        # shortened name to avoid class with type name.
+        #
+        # **NOTE** The Swagger naming does NOT match the names that will be used
+        #          in the final helm values file.  Instead there needs to be an
+        #          explicit mapping which is done in TWO places:
+        #          1. The ConfigDP is able to map the (unprotected) settings but
+        #             does not have access to the protected settings so...
+        #          2. This Powershell script has to perform the mapping for
```

Comment:
This is no longer true and we need to update the comment to reflect the new scheme once we are happy with it.